pages/about/index.md

```diff
@@ -2,22 +2,43 @@
 title: About
 
 ---
+## AsyncAPI
+
+AsyncAPI is an Apache License 2.0 library under the Linux Foundation that seeks to improve the current state of Event-Driven Architectures (EDA). The AsyncAPI Initiative is a specification and growing set of open-source tools to help developers define asynchronous APIs, and build and maintain event-driven architectures. 
+
+Developers familiar with OpenAPI (aka Swagger) for RESTful APIs will see strong similarities when using AsyncAPI. One common use case is generating asynchronous API documentation (HTML or Markdown). The specification is both platform and language agnostic. The current tooling includes support for common message brokers such as Apache Kafka and RabbitMQ and languages including Python, Java, and Nodejs. 
+
+Our long-term goal is to make working with EDAs as easy as working with REST APIs. That goes from documentation to code generation, discovery to event management, and beyond. Our 150+ Open-Source (OSS) contributors are EDA enthusiasts from all around the world.
+
+
+## History
```

Comment:
We need to adjust this whole section about history as it sounds like it represents people working on AsyncAPI -> `We are in love`

It should be in a form "that you describe it as an observer of the history", not "that you describe it as your history".

In other words, instead of this:

```
This project started at Hitch (a.k.a. API Changelog). We are in love with API machine-readable documentation formats (OpenAPI, RAML, etc.) and always encouraged our customers to use a machine-readable definition for their APIs because it allows them to get extra benefits. From documentation and code generation to setup their own [API Assistant](http://apiassistant.com).

Internally, we used a message-driven microservices architecture and we couldn't have all those tools that we have with the HTTP APIs. All those things you get because you have your API defined with a machine-readable documentation format. And, on top of that, we noticed that some of our customers were creating IoT APIs over MQTT and they couldn't get all the benefits from Hitch because the existing specifications don't support message-driven APIs.

**It was time to come up with something good enough for everyone instead of having every company doing its own battle. We can learn a lot together if we join forces, so we decided to create AsyncAPI!**
```

IMHO we should write:

```
AsyncAPI project started at Hitch (a.k.a. API Changelog). They needed a machine-readable format to describe internal message-driven microservices architecture. The goal was to benefit from it in the same way they already used OpenAPI for their HTTP APIs. This is how inspired by OpenAPI, the AsyncAPI specification was created by [Fran Méndez](https://twitter.com/fmvilas) who worked at Hitch at a time. After publishing the first version, he noticed that he solved not only his problems but the problems of the wider community. This is how AsyncAPI specification started evolving into AsyncAPI Initiative, a project driven not by a single man but a much bigger community, under the umbrella of the Linux Foundation.
```

I know it is not the best text and can be written much better. But I hope you know what I mean 😄 

@alequetzalli can you have a look please?

Reply:
Oh yes, I have to say that's true... it reads more like a personal account vs a historical narration. I think what @derberg suggests is better for this case. 

Just a few fixes to his text below, ...also, Fran was starting to sound like our savior 😄✌🏽 hehehehehehe...

```suggestion
The AsyncAPI project started at Hitch (a.k.a. API Changelog). Hitch needed a machine-readable format to describe internal message-driven microservices architecture. The goal was to benefit from it in the same way they had used OpenAPI for their HTTP APIs. Inspired by OpenAPI, [Fran Méndez](https://twitter.com/fmvilas), who worked at Hitch at the time, created the AsyncAPI specification. After publishing the first version, he noticed that he solved not only his problems but the problems of the wider community. Eventually, the AsyncAPI specification and community evolved into the AsyncAPI Initiative, a project now under the umbrella of the Linux Foundation.
```